libkbfs/folder_branch_ops.go

```diff
@@ -950,6 +950,10 @@ func (fbo *folderBranchOps) getMDLocked(
 	if err != nil {
 		return ImmutableRootMetadata{}, err
 	}
+	err = fbo.identifyOnce(ctx, md.ReadOnly())
```

Comment:
Yeah, I don't think this is correct -- if rtype is `mdReadNoIdentify` that's an explicit signal to skip the identify.